addons/website/static/src/snippets/s_product_catalog/002.scss

```diff
@@ -0,0 +1,3 @@
+.s_product_catalog[data-vcss='002'] {
```

Comment:
Add a note explaining why this empty file is necessary ;)

Reply:
Done 👍 What do you think ?